src/pages/Governance/Proposal/card/YourVoteSection.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Section from "./Section";
+import HowToVoteIcon from "@mui/icons-material/HowToVote";
+import {Stack, Typography} from "@mui/material";
+
+// TODO: add the specific vote (yes/no) after indexer is ready
+export default function YourVoteSection(): JSX.Element {
```

Comment:
This component would be more flexible if it accepted `{...rest}` props and then passed them into the Section component (this way you could add a margin to it, for example). It's a good idea to consider any time you're making a component.